lib/resources/service.rb

```diff
@@ -162,7 +162,11 @@ def select_service_mgmt # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticCom
       elsif %w{aix}.include?(platform)
         SrcMstr.new(inspec)
       elsif %w{amazon}.include?(platform)
-        Upstart.new(inspec, service_ctl)
+        if os[:release].start_with?('2.')
```

Comment:
Since 2.x will be the new default, I am curious if we should turn the logic the other way: use systemd as a default for amazon linux and just use upstart if the version string starts with `20\d\d`

Reply:
That's a very good point, I've updated the logic